addok_psql/plugin.py

```diff
@@ -1,4 +1,4 @@
-from addok import config
+from addok.config import config
```

Comment:
Isn't what we just merged? :)

Reply:
Yes, but because it wasn't merged when I made the pull request, I had to get this commit. Will rebase now.

Reply:
Ok, not very familiar with pull requests and rebasing but I think it is OK now.